data/json/recipes/recipe_others.json

```diff
@@ -2043,6 +2043,22 @@
     "autolearn": true,
     "components": [ [ [ "dogbane", 1 ], [ "cattail_stalk", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "plant_fibre",
+    "id_suffix": "s_scutcher",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "survival",
+    "difficulty": 1,
+    "time": "6 m",
+    "batch_time_factors": [ 50, 2 ],
+    "autolearn": true,
+    "tools": [ [ [ "scutcheritem", -1 ] ] ],
+    "components": [ [ [ "dogbane", 8 ], [ "cattail_stalk", 8 ], [ "stick_fiber", 1 ] ] ],
```

Comment:
I don't think the fibrous stalks ("stick_fiber") is used for anything else anymore, is it? Maybe I am not seeing something.

Speaking of plant fiber components, though, I thought of adding bark to the recipe, as having only cattails and dogbane as resources for that seems odd. Since you are working on this related parts anyway, what would you think of adding that change in here as well?
Otherwise I'd make a separate PR where I need to extend your new recipe as well as the manual plant fiber recipe, which seems to be an avoidable extra step - granted you even like the idea of course.

Reply:
Oh, I added the fibrous stalks as some other thing to get fibre from, and you can get them from plant zombies and triffids. Currently they get disassembled to get the fibre. I would have included cannabis plants but the disassembly for that doesn't match the others.
As for bark....I'm not aware of the barks available (birch, willow, alder, tan) in the game being processable into sewing thread-like material? Ropes and cord, yes, but not thread.

What the scutcher furniture actually is, is a pair of hinged boards with ribbing on legs. That part is for breaking up the stalks and separating the bast fibre from the stem core. That job is called just...breaking or something. Scutching is running the resulting mass of fibres through combs to straighten and remove lingering bits of non-fibre stuff.
I just put both things on the same furniture, as its a big custom-built thing. Doing both jobs, just en-mass instead of one stalk at a time.

Reply:
Ah i see, I overlooked that. 
Yeah maybe that doesn't fit with the scutcher then anyway. My thought regarding using bark for making plant fiber was that it is often used in cases where it represents some kind of cordage, strings and stuff like that in a lot of crafting recipes. Plant fiber for this kind of use was, afaik, not only aquired from cattails and dogbane, but also from the softer, inner parts of bark. But maybe it makes more sense to restrict that to the manual method (if at all). I'll maybe make a separate PR then to discuss that further then! ^^